helper/testhelpers/testhelpers.go

```diff
@@ -769,15 +771,15 @@ func SetNonRootToken(client *api.Client) error {
 
 // RetryUntil runs f until it returns a nil result or the timeout is reached.
 // If a nil result hasn't been obtained by timeout, calls t.Fatal.
-func RetryUntil(t testing.T, timeout time.Duration, f func() error) {
+func RetryUntil(t testing.T, timeout, sleepTime time.Duration, f func() error) {
```

Comment:
Why not retain the original method, and factor out this method as a new one, e.g. RetryUntilAtCadence?  That way you won't have to modify all the existing users, which don't need this flexibility.